MSAL/test/unit/native_auth/controllers/MSALNativeAuthCredentialsControllerTests.swift

```diff
@@ -156,15 +159,18 @@ final class MSALNativeAuthCredentialsControllerTests: MSALNativeAuthTestCase {
         factory.mockMakeUserAccountResult(userAccountResult)
         tokenResult.accessToken = MSIDAccessToken()
         tokenResult.accessToken.accessToken = expectedAccessToken
+        tokenResult.rawIdToken = expectedIdToken
         responseValidatorMock.tokenValidatedResponse = .success(tokenResponse)
-        cacheAccessorMock.mockUserAccounts = [account]
+        cacheAccessorMock.mockUserAccounts = [newAccount!]
```

Comment:
can we avoid the force unwrap here?

Reply:
The only point is that if newAccount is nil, we crash, and all the next tests are not executed. While if we gracefully unwrap it only this test will fail